src/main/java/org/shanerx/tradeshop/utils/gsonprocessing/typeadapters/ConfigurationSerializableAdapter.java

```diff
@@ -25,33 +25,36 @@
 
 package org.shanerx.tradeshop.utils.gsonprocessing.typeadapters;
 
-import com.google.gson.JsonDeserializationContext;
-import com.google.gson.JsonDeserializer;
-import com.google.gson.JsonElement;
-import com.google.gson.JsonParseException;
-import com.google.gson.JsonSerializationContext;
-import com.google.gson.JsonSerializer;
+import com.google.gson.*;
```

Comment:
Your Idea is combining imports into wildcards

Reply:
That's actually the IDE, I don't like that